src/Accelerators/NNPA/NNPAAccelerator.hpp

```diff
@@ -0,0 +1,30 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===-------------------------- NNPAAccelerator.hpp
+//-------------------------===//
+//
+// Copyright 2022 The IBM Research Authors.
+//
+// =============================================================================
+//
+// Accelerator class for NNPA
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+#include "src/Accelerators/Accelerator.hpp"
+
+namespace mlir {
+class NNPAAccelerator : public Accelerator {
```

Comment:
[Suggestion]: Make class `final` to prevent deriving it (assuming that NNPA accelerator will not need to be derived from at this point at least).

Reply:
Done